exporter/opentelemetry-exporter-jaeger/src/opentelemetry/exporter/jaeger/__init__.py

```diff
@@ -177,6 +177,22 @@ def shutdown(self):
         pass
 
 
+def _parameter_setter(param, env_variable, default):
```

Comment:
I wonder if this should be in the configuration object to ensure the same order of precedence is used everywhere.